locale/pt-br/texts.po

```diff
@@ -1336,10 +1339,39 @@ msgstr "Ver detalhes"
 msgid "Unregister contract"
 msgstr "Desregistrar contract"
 
+#: src/components/NanoContract/NanoContractTransactionBalanceList.component.js:161
+msgid "No Tokens Balance"
+msgstr "Nenhum saldo de token"
+
+#: src/components/NanoContract/NanoContractTransactionBalanceList.component.js:162
+msgid "The transaction doesn't have token deposit or token withdrawal."
+msgstr "A transação não tem saldo de token para depósito nem para saque."
+
+#: src/components/NanoContract/NanoContractTransactionBalanceListItem.component.js:109
+msgid "Sent ${ tokenValue }"
+msgstr "Enviado ${ symbol }"
+
+#: src/components/NanoContract/NanoContractTransactionBalanceListItem.component.js:110
+msgid "Received ${ tokenValue }"
+msgstr "Recebido ${ symbol }"
```

Comment:
These are outdated, this component doesn't exist anymore

Reply:
Indeed.